private/buf/bufcli/bufcli.go

```diff
@@ -351,13 +351,15 @@ func NewWireImageConfigReader(
 	if err != nil {
 		return nil, err
 	}
+	configProvider := bufconfig.NewProvider(logger)
```

Comment:
nit: we should revert these being split out into parameters

Reply:
Gah, yea I rushed these. Will update.